modules/ding_frontpage/ding_frontpage.pages_default.inc

```diff
@@ -158,6 +161,105 @@ function ding_frontpage_default_page_manager_pages() {
     $display->content['new-18cfc092-1d0f-42e4-bde8-8a6e314c9bd0'] = $pane;
     $display->panels['primary'][1] = 'new-18cfc092-1d0f-42e4-bde8-8a6e314c9bd0';
     $pane = new stdClass();
+    $pane->pid = 'new-5bf05877-cd2f-42d8-92a9-fbe2ef5163ba';
+    $pane->panel = 'primary';
+    $pane->type = 'interaction_pane';
+    $pane->subtype = 'interaction_pane';
```

Comment:
This creates a soft dependency between `ding_frontpage` and P2. Panels will not fail but will show a "panel missing" in the administration interface.